Realm.Shared/linq/RealmResultsVisitor.cs

```diff
@@ -219,9 +206,19 @@ internal override Expression VisitMethodCall(MethodCallExpression m)
                     if (firstRowPtr != IntPtr.Zero)
                         return Expression.Constant(_realm.MakeObjectForRow(_metadata, firstRowPtr));
                     if (m.Method.Name == "First")
-                            throw new InvalidOperationException("Sequence contains no matching element");
+                        throw new InvalidOperationException("Sequence contains no matching element");
                     Debug.Assert (m.Method.Name == "FirstorDefault");
```

Comment:
There's a typo in the method name here. Use `nameof(Queryable._methodName_)` to avoid this.